OrbitLinuxTracing/UprobesReturnAddressManagerTest.cpp

```diff
@@ -390,4 +392,228 @@ TEST(UprobesReturnAddressManager, TailCallOptimization) {
   test_handler.OnNonUretprobesReturn();
 }
 
+//==============================================================================
+// Tests for frame pointer based callchains start here:
+//==============================================================================
+
+const std::string maps_string =
```

Comment:
This should be static or inside an anonymous namespace.

Reply:
I added an anonymous namespace.